### DISCUSSION TITLE
Download Character...

### DISCUSSION THREAD

**Discussion by Crissomar** [Pryaxis/TShock#1713] [General]
tshock.ignore.ssc is already there but my character doesn't update...
I enter the server with SSC activated, I get the items, I give myself tshock.ignore.ssc permission but nothing...
i also enter the server with tshock.ignore.ssc permission and then I remove it its like uploading the character. Just help me quiting that SSC flag which doesn't let my character update in client when tshock.ignore.ssc permission is on.

**Comment by bartico6** [comment]
If you're using an admin account with * permissions, you need to grant yourself `!tshock.ignore.ssc`. Note the exclamation point, it's a negation, it *denies* the permission that would otherwise be granted by *. If you're using a superadmin account, you need to create a new permission group that will have the tshock,ignore.ssc permission **negated**. If you give yourself the permission w/o the negation, the server won't save whatever changes you make to your character on purpose.

Also note that you won't see changes in your local (client) character - the point of SSC is to ensure changes are not made. It's a design choice in Terraria's ecosystem.

**Comment by bippity** [comment]
Also see
https://github.com/Pryaxis/TShock/discussions/1697#discussioncomment-4538
